### DISCUSSION TITLE
Multiple IP addresses - Standalone mode setup

### DISCUSSION THREAD

**Discussion by jasongomes** [Security-Onion-Solutions/securityonion#12070] [Unsupported Versions] [answered]
I'm not able to add more than one subnet in STANDALONE setup. It gives an "Invalid input" error.
 
![cidr notation](https://github.com/Security-Onion-Solutions/securityonion/assets/5719043/2818aa5a-c4c4-4bad-b63a-742e9609593e)

**Comment by TOoSmOotH** [comment]
It only accepts a single range. You can add the additional after setup completes. Just make sure you put in the range that you are accessing it from first.

**Reply by jasongomes** [reply]
I have installed it. Next,where do i input the remaining subnets? in the web interface or  ssh into the server and make changes there? if so then where??

**Reply by TOoSmOotH** [reply, marked as answer]
https://docs.securityonion.net/en/2.4/firewall.html
